docs/01.GETTING-STARTED.md

```diff
@@ -81,6 +81,36 @@ For example the cross-compile to RaspberryPi 2 is something like this:
 python tools/build.py --toolchain=cmake/toolchain_linux_armv7l.cmake
 ```
 
+**Use system memory allocator**
+
+```bash
+python tools/build.py --system-allocator=on --jerry-libc=off
```

Comment:
I think it would be good to add `--compile-flag=-m32` to the command or add some hints about how to build to 32bit.